app/views/previous_points/manual_points.html.erb

```diff
@@ -0,0 +1,34 @@
+<h5><%= link_to("Back to List", points_path, :class => 'back-link') %></h5>
+
+<% @page_title = "Member Manual Points" %>
+<div class="member points index">
+<h3>Member Points</h3>
+
+<p><i> Shows all the manual points you have earned</p></i>
+
+<p>
+	<strong> Manual Points : </strong>
+	<%= @member.manual_points_received(true) %>
+	<br/>
+	<strong> All Time Manual Points : </strong>
+	<%= @member.manual_points_received %>
+</p>
+
+<table class="table" summary="Manual Points List">
+
+    <tr class="header">
+    <th>Points</th>
+		<th>Reason</th>
+		<th>Category</th>
+		<th>When</th>
+    </tr>
+    <% @member.manual_points.where(:semester_id == @sem_id).each do |manual_points| %>
```

Comment:
here too

Reply:
This is not ranges, but I changed it to the correct syntax.